src/components/Modal/Modal.vue

```diff
@@ -154,13 +220,32 @@ export default {
 				// swiping to right to go back to the previous item
 				this.previous(e)
 			}
+		},
+		handleMouseMove() {
+			this.clearView = false
+			clearTimeout(this.clearViewTimeout)
+			this.clearViewTimeout = setTimeout(() => {
+				this.clearView = true
+			}, 5000)
+		},
+		handleSlideshow() {
+			this.playing = true
+			if (this.hasNext) {
+				this.slideshowTimeout = setTimeout(() => {
+					this.next()
+					this.handleSlideshow()
+				}, 3000)
```

Comment:
It would be nice to have this dynamic :wink:

Reply:
you mean able to choose the time?

Reply:
Yes.